pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/JsonAsyncHttpPinotClientTransport.java

```diff
@@ -47,6 +47,9 @@
   private final Map<String, String> _headers;
   private final String _scheme;
 
+  private static final long BROKER_READ_TIMEOUT = 1200000L;
```

Comment:
20 minutes timeout seems too long. I feel 1 minute might be enough?

Reply:
Also suggest appending `_MS` to be more clear (`BROKER_READ_TIMEOUT_MS` and `BROKER_CONNECT_TIMEOUT_MS`)